src/main/resources/openapi/schemas/ingestJobRecordResponse.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "Ingest job record response",
+  "type": "object",
+  "properties": {
+    "globalId": {
+      "type": "string",
+      "description" : "Global identifier",
+      "format" : "uuid"
+    },
+    "matchKeys": {
+      "type": "object"
```

Comment:
Can we constrain the type of object keys to string -> array

Reply:
I tried hard on this with patternProperties.. Spent 30 minutes and getting no where. It just wont parse. I'll push the version I have for this ingestJobRecordResponsePattern.json

Reply:
```suggestion
      "type": "object",
      "patternProperties" : {
        "^.*" : {
          "type": "array"
        }
      },
      "additionalProperties" : true
```

Reply:
Oh... I forget again OpenAPI is *not* JSON schema.. Many things aren't supported and other things are OpenAPI specific.  https://swagger.io/docs/specification/data-models/keywords/